src/modals/PostModal/Styles.ts

```diff
@@ -9,3 +9,12 @@ export const Bumper = styled.div`
 export const Modal = styled(UModal)`
   width: 360px;
 `;
+export const InputBox = styled(Textarea)`
```

Comment:
We can keep the name `Textarea` instead of `InputBox`?
```suggestion
export const Textarea = styled(UTextarea)`
```